lib/foreman_maintain/config.rb

```diff
@@ -1,17 +1,31 @@
+require 'fileutils'
 module ForemanMaintain
   class Config
-    attr_accessor :definitions_dirs, :log_level
+    attr_accessor :definitions_dirs, :log_level, :log_dir
 
     def initialize(options = {})
       @definitions_dirs = options.fetch(:definitions_dirs,
                                         [File.join(source_path, 'definitions')])
+
       @log_level = options.fetch(:log_level, ::Logger::ERROR)
```

Comment:
Since we are logging in to a file now, I think it's benefitial to have debug mode turned on by default: we can change thsi later, but for this stage, having as much info as possible from the run would be benefitial, unless you think it could bite us somewhere?